qa/tasks/mgr/dashboard/test_auth.py

```diff
@@ -31,14 +32,14 @@ def _validate_jwt_token(self, token, username, permissions):
 
     def test_a_set_login_credentials(self):
         self.create_user('admin2', 'admin2', ['administrator'])
-        self._post("/api/auth", {'username': 'admin2', 'password': 'admin2'})
+        self._post("/api/auth", {'username': 'admin2', 'password': 'admin2'}, version=DEFAULT_VERSION)
```

Comment:
I'm wondering if we really need to specify `version=DEFAULT_VERSION` on all these `_post`, `_delete`, `_put` and `_get` calls as we already defined `DEFAULT_VERSION` as default value in helpers.py .

Reply:
Totally agree on removing it! It was there, as initially version=None in helper.py, in order to make it useful to set the version accordingly in the test files, but it's no more the case, so makes sense to remove it.
Thanks @s0nea for pointing it